packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -562,7 +564,7 @@ class DartIOHttpRequestData extends HttpRequestData {
 
   /// All instant events logged to the timeline for this HTTP request.
   @override
-  List<HttpInstantEvent> get instantEvents {
+  List<HttpInstantEvent>? get instantEvents {
```

Comment:
I think this can be non-null. `_instantEvents` is assigned in the getter body

Reply:
done